cri/v1alpha2/cri.go

```diff
@@ -306,21 +333,8 @@ func (c *CriManager) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 		}()
 	}
 
-	sandboxMeta := &SandboxMeta{
-		ID:        id,
-		Config:    config,
-		NetNSPath: netnsPath,
-		Runtime:   config.Annotations[anno.KubernetesRuntime],
-	}
-
-	if _, ok := config.Annotations[anno.LxcfsEnabled]; ok {
-		enableLxcfs, err := strconv.ParseBool(config.Annotations[anno.LxcfsEnabled])
-		if err != nil {
-			return nil, err
-		}
-		sandboxMeta.LxcfsEnabled = enableLxcfs
-	}
-
+	// update the metadata of sandbox container after network had been set up successfully.
+	sandboxMeta.NetNSPath = netnsPath
```

Comment:
I think this part should be following `c.setupPodNetwork`. If there is no need to setup Pod network, are we need to update the NetNSPath here?